Sniff.php

```diff
@@ -658,7 +664,7 @@ protected function is_safe_casted( $stackPtr ) {
 
 		// Get the last non-empty token.
 		$prev = $this->phpcsFile->findPrevious(
-			PHP_CodeSniffer_Tokens::$emptyTokens
+			Tokens::$emptyTokens
 			, $stackPtr - 1
 			, null
```

Comment:
(Realize it's not new/not yours but...) there's an almost entertaining quality about seeing this particular coding style with commas at the start of each line within a code sniffer :smile: